app/scienceapi/static/admin/custom-event-admin.css

```diff
@@ -0,0 +1,17 @@
+/* Removes Timezone warning from admin */
+.timezonewarning {
+	display: none;
+}
+
+/* Remove relative datetime pickers */
+.datetimeshortcuts > * {
```

Comment:
Let's keep the clock visible, it's useful because it shows what format the time needs to be in